### DISCUSSION TITLE
无法传送文件给平板

### DISCUSSION THREAD

**Discussion by rygdsddssd** [localsend/localsend#1050] [L-简体中文]
![image](https://github.com/localsend/localsend/assets/86097988/a003aa51-fcbd-47d1-b4cc-84b26409ab05)
[x]平板端已授予权限
平板可以发文件给电脑；电脑、手机都无法给平板发送文件

**Comment by specked2527** [comment]
使用通过链接传送

**Reply by rygdsddssd** [reply]
使用平板打开链接一直“等待响应”

**Reply by rygdsddssd** [reply]
成功了
